packages/blade/src/components/Button/BaseButton/StyledBaseButton.native.tsx

```diff
@@ -56,6 +58,8 @@ const StyledBaseButton = ({
 
   return (
     <StyledPressable
+      {...props}
```

Comment:
We have to maybe assess this. 

StyledPressable is the native platform component which will need all the props for accessibility. like `accessibilityLabel`, `accessibilityRole` if we don't spread it it will not be passed to the underline platform. 

We can't also say that just accept `accessibilityLabel` or `accessibilityRole` because now we will also have to accept `role` & `aria-label` separately for web.

Reply:
Discussed, we'll accept `accessibilityProps` for StlyedBaseButton and only spread that here